xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -138,13 +138,13 @@ extern "C"
       TYPE_AUDIO,
```

Comment:
Set the to `TYPE_NONE = 0,` to confirm is really always 0, also on addon side.

Reply:
Done